docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+Check the LOCATION printed below if you are unsure where to go.
+
+Oh... and here are some vitals:
+
+ NAME      <project name>
+ LOCATION  /Users/<your user>/<path to your project>
+ RECIPE    platformsh
+ DOCS      https://docs.lando.dev/config/platformsh.html
+```
+
+Once it completes, run `lando start` to start your app and services.
+
+``` bash
 $ lando start
+Let's get this party started! Starting app <app name>...
+...
 ```
 
-You can then download data from the environment for your current branch and load it into your local Lando environment by running:
+Once the command ran successfully, you get the following:
+
+``` bash
+
+   ___                      __        __        __     __        ______
+  / _ )___  ___  __ _  ___ / /  ___ _/ /_____ _/ /__ _/ /_____ _/ / / /
+ / _  / _ \/ _ \/  ' \(_-</ _ \/ _ `/  '_/ _ `/ / _ `/  '_/ _ `/_/_/_/
+/____/\___/\___/_/_/_/___/_//_/\_,_/_/\_\\_,_/_/\_,_/_/\_\\_,_(_|_|_)
 
-```bash
-lando pull -r database -m web/sites/default/files
+
+Your app has started up correctly.
+Here are some vitals:
+
+ NAME          <project name>
+ LOCATION      /Users/<your user>/<path to your project>
+ SERVICES      <list of your service's names>
+ APP URLS      http://localhost:62177
+               http://<app name>.lndo.site/
+               https://<app name>.lndo.site/
+               http://www.<app name>.lndo.site/
+               https://www.<app name>.lndo.site/
 ```
 
-Where the value after `-r` is the relationship name in the application for the service you want to download, and `-m` is the Platform.sh mount path you want to download.  You may specify as many relationships (multiple `-r` switches) and as many mounts (multiple `-m` switches) as you wish.
+Access your app and services by opening the `APP URLS` in your browser.
+
+For more examples, see the [getting started page](https://docs.lando.dev/platformsh/getting-started.html).
+
+## Configuring lando
+
+Lando configuration is located in [the `.lando.yml` file](https://docs.lando.dev/platformsh/config.html).
+
+If you changed your [`.platform.app.yaml`](../../configuration/app/_index.md), [`.platform/routes.yaml`](../../configuration/routes/_index.md) configuration files, run `lando rebuild` for the changes to be taken into account.
+
+### Untrusted SSL certificate
+
+By default, when accessing your local Lando sites through `https` you get an error message in your browser.
+This is expected behavior:
+
+> Lando uses its own Certificate Authority (CA) to sign the certificates for each service and to ensure that these certs are trusted on the internal Lando network.
+However, while Lando will automatically trust this CA internally it's up to you to trust it on your host machine. Doing so alleviates browser warnings regarding the certificates issued.
+
+Find out how to solve it in [Lando's Blog](https://lando.dev/blog/2020/03/20/_5-things-to-do-after-you-install-lando/).
+
+## Common tasks
+
+**TODO: clarify that part + test**
```

Comment:
Why is it necessary to test? What needs to be clarified?

I feel like a lot of this is just part of a what's next section that we don't need to feel responsible for at all.